appinventor/docs/reference/components/userinterface.html

```diff
@@ -1156,6 +1156,10 @@ <h2 id="Switch"> Switch </h2>
                     their
                     boolean state in response. They are identical to Checkboxes except in appearance.</p>
                   <p> Switches have an on (true) state and an off (false) state. A switch component raises an event when the user taps it to toggle between states. </p>
+                  <p> Switches does not work on Android versions < 4.0. So Switch will automatically get
+                      converted to a Checkbox on a devices older than SDK 14.<br> 
```

Comment:
Grammar corrections:
"Switches do not work"
"Switch will be automatically converted"
"on devices older than SDK 14."

Thanks!

Reply:
@SusanRatiLane Thanks for pointing out the errors. I fixed them. Please have a look.

Thanks!!